mm2src/mm2_main/src/rpc/lp_commands/one_inch/types.rs

```diff
@@ -213,13 +225,13 @@ pub struct TxFields {
 impl TxFields {
     pub(crate) fn from_api_tx_fields(
         tx_fields: one_inch_api::classic_swap_types::TxFields,
-        decimals: u8,
+        dst_decimals: u8,
```

Comment:
but the parameter here is called `dst_decimals`
is that wrong or there is some reasoning of why src in one plac changes to dst in another?

Reply:
fixed thanks (ofc src_decimals)